src/Symfony/Component/Mailer/Bridge/Sendinblue/composer.json

```diff
@@ -0,0 +1,37 @@
+{
+    "name": "symfony/sendinblue-mailer",
+    "type": "symfony-bridge",
+    "description": "Symfony Sendinblue Mailer Bridge",
+    "keywords": [],
+    "homepage": "https://symfony.com",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Fabien Potencier",
+            "email": "fabien@symfony.com"
+        },
+        {
+            "name": "Symfony Community",
+            "homepage": "https://symfony.com/contributors"
+        }
+    ],
+    "require": {
+        "php": ">=7.2.5",
+        "symfony/mailer": "^4.4|^5.0"
```

Comment:
Should be ^5.1 (as you are using MetadataHeader which is not part of earlier versions)

Reply:
Done.